contracts/SavingAccount.sol

```diff
@@ -229,4 +235,14 @@ contract SavingAccount is Initializable, InitializableReentrancyGuard, Constant,
 
         emit Claim(msg.sender, FINAmount);
     }
+
+    /**
+     * Withdraw COMP token to beneficiary
+     */
+    function withdrawCOMP(address _beneficiary) external onlyOwner {
+        uint compBalance = IERC20(COMP_ADDR).balanceOf(address(this));
+        IERC20(COMP_ADDR).safeTransfer(_beneficiary, compBalance);
```

Comment:
I don't think that this approach is the right way to claim the COMP tokens because of the following reasons:

1. `safeTransfer` is being executed on the hard coded mainnet COMP address. This won't allow us to properly test this feature.
2. Compound Protocol offers a `claimCOMP` function which they recommend should be called by users to claim their accrued COMP from different markets supported on their platform. As detailed here: https://compound.finance/docs/comptroller#claim-comp
3. A better approach would be to create a function in the contracts, that is able to call this `claimCOMP` function which will also allow us to test this functionality as well.

Reply:
1. We can test that by changing the COMP_ADDR in test env.
2. We dont need to call `claimCOMP()` function on Compound as the function can be called on-behalf-of as well.
3. Same point as 2.